valhalla/jawn/src/controllers/public/propertyController.ts

```diff
@@ -1,90 +1,155 @@
 import {
-  Body,
-  Controller,
-  Path,
-  Post,
-  Request,
-  Route,
-  Security,
-  Tags,
+	Body,
+	Controller,
+	Path,
+	Post,
+	Request,
+	Route,
+	Security,
+	Tags,
 } from "tsoa";
-import { KVCache } from "../../lib/cache/kvCache";
 import { dbQueryClickhouse } from "../../lib/shared/db/dbExecute";
+import { clickhouseDb } from "../../lib/db/ClickhouseWrapper";
 import { buildFilterWithAuthClickHouse, buildFilterWithAuthClickHouseOrganizationProperties } from "@helicone-package/filters/filters";
 import { resultMap } from "../../packages/common/result";
 import type { JawnAuthenticatedRequest } from "../../types/request";
-import { quickCacheResultCustom } from "../../utils/cacheResult";
 import { COST_PRECISION_MULTIPLIER } from "@helicone-package/cost/costCalc";
+import { getFromCache, storeInCache, clearCache } from "../../lib/cache/staticMemCache";
 
 export interface Property {
-  property: string;
+	property: string;
 }
 
 export interface TimeFilterRequest {
-  timeFilter: {
-    start: string;
-    end: string;
-  };
+	timeFilter: {
+		start: string;
+		end: string;
+	};
 }
 
-const longCache = new KVCache(60 * 60 * 1000 * 7); // 1 week
+// Properties queries are cached per-org via staticMemCache (see below)
 
 @Route("v1/property")
 @Tags("Property")
 @Security("api_key")
 export class PropertyController extends Controller {
-  @Post("query")
-  public async getProperties(
-    @Body()
-    requestBody: {},
-    @Request() request: JawnAuthenticatedRequest
-  ) {
-    const builtFilter = await buildFilterWithAuthClickHouseOrganizationProperties({
-      org_id: request.authParams.organizationId,
-      argsAcc: [],
-      filter: {},
-    });
-
-    const query = `
-    SELECT DISTINCT property_key AS property
+	@Post("query")
```

Comment:
can you update your local to use the same prettier that is used for us?

really only tabs to spaces. and then 2 spaces per tab or something like that... we'd hate to keep getting irrelevant diffs like these